config.go

```diff
@@ -14,7 +14,8 @@ type config struct {
 	Listen      string
 	ListenPprof string `yaml:"listen_pprof"`
 
-	Target string
+	Target      string
+	SupportIPv6 bool `yaml:"support_ipv6"`
```

Comment:
Arguably `enable` would fit better than `support`, i.e.:
```suggestion
	EnableIPv6 bool `yaml:"enable_ipv6"`
```

Reply:
Changed name to `EnableIPv6` and `enable_ipv6`